decidim-core/app/uploaders/decidim/homepage_image_uploader.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+module Decidim
+  # This class deals with uploading hero images to ParticipatoryProcesses.
+  class HomepageImageUploader < ApplicationUploader
```

Comment:
Should we resize the image somehow? Also, have a look at `AttachmentUploader` and `ParticipatoryProcessAttachment` on how to handle validations and file types.